limitador/src/storage/redis/redis_async.rs

```diff
@@ -231,6 +233,30 @@ impl AsyncRedisStorage {
 
         Ok(())
     }
+
+    pub async fn update_counters(
```

Comment:
this fn could be named differently, a more explicit name would be `batch_update_counters`... will leave it to the review